specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -79,13 +82,27 @@
             "$ref": "#/parameters/ApiVersionParameter"
           },
           {
-            "$ref": "#/parameters/AutocompleteModeParameter"
+            "name": "autocompleteMode",
+            "in": "query",
+            "required": true,      
+            "type": "string",
+            "enum": [
+              "oneTerm",
+              "twoTerms",
+              "oneTermWithContext"
+            ],
+            "x-ms-enum": { 
+             "name": "AutocompleteMode",
+             "modelAsString": false
+            },
+            "x-ms-parameter-location": "method",
```

Comment:
I don't think you need `x-ms-parameter-location` for parameters that are inline with the operation.